projects/plugins/jetpack/modules/masterbar/nudges/additional-css/class-atomic-additional-css-manager.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * WPORG_Additional_CSS_Manager file
+ *
+ * Responsible with replacing the Core Additional CSS section with an upgrade nudge on Atomic.
+ *
+ * @package Jetpack
+ */
+
+namespace Automattic\Jetpack\Dashboard_Customizations;
+
+/**
+ * Class Atomic_Additional_CSS_Manager
+ *
+ * @package Automattic\Jetpack\Dashboard_Customizations
+ */
+class Atomic_Additional_CSS_Manager {
+
+	/**
+	 * The site domain.
+	 *
+	 * @var string
+	 */
+	private $domain;
+
+	/**
+	 * Atomic_Additional_CSS_Manager constructor.
+	 *
+	 * @param string $domain the Site domain.
+	 */
+	public function __construct( $domain ) {
+		$this->domain = $domain;
+	}
```

Comment:
Can `Atomic_Additional_CSS_Manager` extend from `WPCOM_Additional_CSS_Manager` so can get rid of this constructor?

Reply:
That would add a bit of coupling between these two classes. Plus, we would need load both classes for AT, even though most of that logic is not needed in there.